src/database/database.ts

```diff
@@ -21,8 +25,6 @@ export const insertVtBooking = async function (
   vehicleBooking.FK_VEH_SYST_NO = 1234567;
```

Comment:
looks like the value of `FK_VEH_SYST_NO` will always be 1234567, maybe worth adding that to the `VehicleBooking.ts`

Reply:
Sorry, I should have mentioned it in the description. There are 4 fields that will be populated via a lookup on the VEHICLE table. That will be done in a separate ticket.